modules/dnn/src/net_impl.cpp

```diff
@@ -613,6 +613,18 @@ void Net::Impl::allocateLayers(const std::vector<LayerPin>& blobsToKeep_)
 
     layersTimings.resize(lastLayerId + 1, 0);
     fuseLayers(blobsToKeep_);
+
+    for (MapIdToLayerData::const_iterator it = layers.begin(); it != layers.end(); it++)
+    {
+        int lid = it->first;
+        LayerData &ld = layers[lid];
+        if (ld.skip)
+        {
+            continue;
+        }
+        Ptr<Layer>& currLayer = ld.layerInstance;
+        currLayer->afterFuse();
```

Comment:
let's try to avoid this part. It's yet another method to think about. Let's keep the protocol simple